ui/src/app/store/bootresource/selectors.test.ts

```diff
@@ -0,0 +1,32 @@
+import bootResourceSelectors from "./selectors";
+
+import {
+  bootResourceState as bootResourceStateFactory,
+  bootResourceStatuses as bootResourceStatusesFactory,
+  rootState as rootStateFactory,
+} from "testing/factories";
+
+describe("bootresource selectors", () => {
+  it("can get all items", () => {
```

Comment:
should this read "can get all statuses"?

Reply:
What?! OK, I'll do it.